capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    puts @nameable.correct_name.capitalize
```

Comment:
- [ ] We should return this capitalized word, not print it. Kindly fix that.

Reply:
ok, fixed
Thanks